scrapli_community/nokia/sros/nokia_sros.py

```diff
@@ -102,5 +128,15 @@
                 "Bad Command:",
             ],
         },
+        "aram": {
+            "privilege_levels": CLASSIC_ARAM_PRIVILEGE_LEVELS,
+            "sync_on_open": classic_aram_sync_on_open,
+            "async_on_open": classic_aram_async_on_open,
+            "timeout_ops": 300,
```

Comment:
is this strictly required? generally I wouldn't want to change the defaults for this here. but... if the dslams are just ultra slow and its required its cool.

Reply:
ok, ill think on this one a bit on how I wanna handle it. Im not a huge fan of changing the timeouts since that may be unexpected for users, but I hear what you're saying. will sleep on it :)

Reply:
I thought not to change it there, and change it instantiating the Scrapli class. But i also though on the users. They could face timeout exception using this variant and could take sometime to identify that the operation have a default timeout of 30 secs and the device is taking longer to finish.

Also to help you decide. These ARAM devices are legacy XDSL, out of market. They all will behave slow. In most of the cases users will have to change this timeout, so why not modify this variant default.